src/solve/slg/implementation/resolvent.rs

```diff
@@ -300,10 +305,10 @@ impl<'t> AnswerSubstitutor<'t> {
     /// case.
     fn assert_matching_vars(&mut self, answer_depth: usize, pending_depth: usize) -> Fallible<()> {
```

Comment:
I think I fixed a bug just below (overflow when substracting the two `usize` args). I don't know why this did not trigger earlier though.

Reply:
curious